p2p/net/conn/listen.go

```diff
@@ -20,6 +22,26 @@ import (
 const SecioTag = "/secio/1.0.0"
 const NoEncryptionTag = "/plaintext/1.0.0"
 
+const connAcceptBuffer = 32
+const NegotiateReadTimeout = time.Second * 20
```

Comment:
Note; this is *just* the timeout for reading the multistream headers from the client. 

If you cant connect and send '/multistream/1.0.0\n/secio/1.0.0\n' in under 60 seconds, youre gonna have a bad time.

I will raise this to 60 seconds.